src/ng/directive/select.js

```diff
@@ -6,8 +6,9 @@ function chromeHack(optionElement) {
   // Workaround for https://code.google.com/p/chromium/issues/detail?id=381459
   // Adding an <option selected="selected"> element to a <select required="required"> should
   // automatically select the new element
-  if (optionElement[0].hasAttribute('selected')) {
-    optionElement[0].selected = true;
+  var optionNode = optionElement[0];
+  if (optionNode.hasAttribute && optionNode.hasAttribute('selected')) {
```

Comment:
Can we instead filter comment nodes in the addOption function? They pollute the optionsMap. Doesn't really do anything, but it's cleaner.